CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Bug fixes
+
+* Fix incorrect floor number increment for Cull on chest/boss fire/boss floors
```

Comment:
You're more than welcome to append your own preferred name here :)

Reply:
I added myself to credits.md on https://github.com/dbjorge/jorbs-spire-mod/pull/557, it's just not merged yet.

Reply:
I'll add my name to the changelog as well